keras_contrib/applications/resnet.py

```diff
@@ -366,20 +513,22 @@ def ResNet(input_shape=None, classes=10, block='bottleneck', residual_unit='v2',
     else:
         residual_unit = residual_unit
 
-    # Permute dimension order if necessary
-    if K.image_data_format() == 'channels_first':
-        input_shape = (input_shape[1], input_shape[2], input_shape[0])
-    # Determine proper input shape
-    input_shape = _obtain_input_shape(input_shape,
-                                      default_size=32,
-                                      min_size=8,
-                                      data_format=K.image_data_format(),
-                                      require_flatten=include_top)
+    # Determine proper input shape again
+    input_shape = obtain_input_shape(input_shape=input_shape, require_flatten=include_top,
```

Comment:
No need to "obtain_input_shape" again. That was included in the original file because there was a permutation of dim ordering before this, which you deleted.